environment.yml

```
@@ -1,11 +1,13 @@
 name: cadquery
 channels:
   - conda-forge
+  - cadquery/dev
   - cadquery
 dependencies:
   - python>=3.8
   - ipython
-  - ocp=7.7.*
+  - ocp=7.7.1.*
+  - python-gmsh
```

```suggestion
  - python-gmsh
  - gmsh
```

I'm not sure about this but I think python -gmsh includes in binding while gmsh is the executable

---

Not needed, gmsh is already a dep of python-gmsh